app/controllers/nonprofits/recurring_donations_controller.rb

```diff
@@ -33,8 +33,8 @@ def export
 			end
 
 			[:active_and_not_failed, :active, :failed].each do |k|
-				if (params.key?(k))
-					params[k] = !!params[k].blank?
```

Comment:
I think I realized what I was trying to do: if `params[k]` was not blank, the params[k] should be set to true. I think I put an extra ! there. I'm gonna confirm that was the issue, right now.

Reply:
I thought so too, but if the key exists, it is always true because the value is there. There was one case that the value came as `"false"`, so what I understood was that you wanted to convert the string to a boolean.

Reply:
Maybe? Are there cases where the value wasn't being sent as "true" or "false"?

Reply:
I understand now I think. Your solution is correct!